InternTrack.Core.Api/Services/Foundations/Interns/IInternService.cs

```diff
@@ -11,5 +12,6 @@ namespace InternTrack.Core.Api.Services.Foundations.Interns
     public interface IInternService
     {
         ValueTask<Intern> AddInternAsync(Intern intern);
+        IQueryable<Intern> RetrieveAllInternAsync();
```

Comment:
```suggestion
        IQueryable<Intern> RetrieveAllInternAsync();
```IQueryable<Intern> RetrieveAllInternsAsync();